src/Bynder/Api/Impl/OAuth2/RequestHandler.php

```diff
@@ -43,7 +43,14 @@ protected function sendAuthenticatedRequest($requestMethod, $uri, $options = [])
             $this->oauthProvider->getAuthenticatedRequest(
                 $requestMethod, $uri, $this->configuration->getToken()
             ),
-            array_merge($options, $this->configuration->getRequestOptions())
+            array_merge(
+                $options,
+                $this->configuration->getRequestOptions(),
+                ['headers'=> [
+                        'User-Agent' => 'bynder-php-sdk/' . $this->getSdkVersion()
```

Comment:
How does this work? Since `getSdkVersion()` is defined in `Configuration`, I'd expect `$this->configuration->getSdkVersion()`?

(and same goes up for the perm tokens implementation)

Reply:
Good catch! This was fixed.